build-livecd-root

```diff
@@ -81,7 +81,7 @@ EOF
 sha256sum initrd0.img vmlinuz0 > SHA256SUM
 
 cd ..
-tarball=$srcdir/fdi-image-${last_tag#release-}.tar
+tarball=$destdir/fdi-image-${last_tag#release-}.tar
 tar cf $tarball fdi-image/
```

Comment:
I added second parameter to this script which allows you to set OUTPUT to different directory.